src/ManageCourses.Api/Model/BackendRequest.cs

```diff
@@ -0,0 +1,7 @@
+namespace GovUk.Education.ManageCourses.Api.Model
+{
+    public class BackendRequest
```

Comment:
add code doc to the class explaining why we need this

Reply:
More verboseness?

Its simply a dto for complex type to serve the mvc frameframe in order to understand that the body of the json that it receives needs to map to the class property.

ie everything that is under 
GovUk.Education.ManageCourses.Api.Model namespace

Reply:
It would make the PR easier to review and the code easier to understand in the future. Up to you if you're not keen.

minor.

Reply:
Not keen